Executables/Migrator/Migrations/20201102194100_CreatePlayersTable.cs

```diff
@@ -11,6 +11,7 @@ public override void Up()
             Create.Table("players")
                 .WithColumn("Id").AsAnsiString(36).PrimaryKey()
                 .WithColumn("AccountId").AsAnsiString(36).PrimaryKey()
+                .WithColumn("Empire").AsByte().WithDefaultValue(0)
```

Comment:
You should create a new migration instead of modifying an existing one. People are forced to roll back their database. This is currently fine but a no-go in the future